fluentd/run.sh

```diff
@@ -105,5 +105,8 @@ else
     echo > $CFG_DIR/dynamic/es-ops-copy-config.conf
 fi
 
-
-exec fluentd $fluentdargs
+if [[ $DEBUG ]] ; then
+    exec fluentd $fluentdargs > /var/log/fluentd.log 2>&1
```

Comment:
@ewolinetz This doesn't work:
```
output='-vv > log 2>&1'
...
exec fluentd $output
```
This does work, but we have to use eval:
```
eval "exec $output"
```
I suppose we could do something like this too:
```
exec fluentd > $stdout 2> $stderr
```
and set `stdout=/dev/stdout` and `stderr=/dev/stderr` by default.  Not sure if `stdout=/var/log/fluentd.log` and `stderr=/var/log/fluentd.log` will work as expected.  I can do some file descriptor magic.